tegel/src/components/stepper/sdds-stepper.tsx

```diff
@@ -1,14 +1,25 @@
-import { Component, Host, h, Prop, Element } from '@stencil/core';
-import { HostElement, State } from '@stencil/core/internal';
+import { Component, Host, h, Prop, Element, Event } from '@stencil/core';
+import { EventEmitter, HostElement, State, Watch } from '@stencil/core/internal';
+type Props = {
```

Comment:
Nice! Question, were not using interfaces at all right? Could also be a way of solving this.
```suggestion
interface SddsStepperProps {
  direction: 'horizontal' | 'vertical';
  labelPosition: 'aside' | 'below';
  size: 'sm' | 'lg';
  hideLabels: boolean;
}
```